Userland/Libraries/LibVideo/PlaybackManager.h

```diff
@@ -180,11 +180,13 @@ class PlaybackManager {
 
     OwnPtr<PlaybackTimer> m_decode_timer;
 
-    NonnullOwnPtr<PlaybackStateHandler> m_playback_handler;
+    OwnPtr<PlaybackStateHandler> m_playback_handler;
     Optional<FrameQueueItem> m_next_frame;
 
     u64 m_skipped_frames { 0 };
 
+    int m_id { 0 };
```

Comment:
Accidentally added / wrong commit?

Reply:
Leftover from debugging, ty for catching that!